src/pages/quote/quote.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Navbar from '../../components/navbar/navbar';
+
+function Quote() {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.